frontend/src/scenes/actions/ActionEdit.js

```diff
@@ -7,6 +7,7 @@ import { actionEditLogic } from './actionEditLogic'
 import { ActionStep } from './ActionStep'
 import { Input } from 'antd'
 
+// TODO: isEditor === false always
```

Comment:
👍 This is great for now as if everything goes well with #1841, this file is already deprecated.